app/views/CreditAndRefunds.scala.html

```diff
@@ -82,162 +64,199 @@
     }
 }
 
+@creditAndRefundRefundProgressHtml(amount: BigDecimal) = {
+    @if(messages.lang.language.toLowerCase().equals("cy")) {
+        <li>
+            <p class="govuk-heading-s">
```

Comment:
* not sure if this is really required to handle language specific scenarios separately;
* have a look at the example how next resource is handled, 'inYear-tax-overview.caption', you should be able to pass parameter in the resource etc;

Reply:
I'm not touch this code at all

Reply:
ok, this can be covered by existing story raised: https://jira.tools.tax.service.gov.uk/browse/MISUV-4059